pytorch_lightning/plugins/ddp_plugin.py

```diff
@@ -108,3 +108,26 @@ def on_before_forward(self, model, *args):
 
     def optimizer_state(self, optimizer: Optimizer) -> dict:
         return optimizer.state_dict()
+
+    def reference_model_in_plugin_wrapper(
```

Comment:
```suggestion
    def cast_model_from_plugin(
```
may we add something about what is the function doing in its name?